pennylane_aqt/device.py

```diff
@@ -206,9 +207,17 @@ def _apply_operation(self, operation):
                 if bit == 1:
                     self._append_op_to_queue("RX", np.pi, wires=[wire])
             return
-
         if op_name == "Hadamard":
-            op_name = "RY"
+            if inv:
+                self._append_op_to_queue("RY", np.pi / 2, wires)
+                self._append_op_to_queue("RX", np.pi, wires)
+            else:
+                self._append_op_to_queue("RX", np.pi, wires)
+                self._append_op_to_queue("RY", -np.pi / 2, wires)
+            return
+
+        if op_name == "S":
+            op_name = "RZ"
```

Comment:
We should add this to PennyLane core as the decomposition of the S gate, so that plugins aren't forced to implement it if not natively supported

Reply:
Agreed, I looked there initially and realized it was not yet implemented

Reply:
https://github.com/XanaduAI/pennylane/issues/644